test/e2e_env/kubernetes/gateway/gateway.go

```diff
@@ -41,7 +41,7 @@ spec:
       tls:
         mode: TERMINATE
         certificates:
-        - secret: example-kuma-io-certificate
+        - secret: kuma-io-certificate-k8s
```

Comment:
This was a weird one, please check if my reasoning is correct. Now that we're removing the secrets using owner reference and we're relying on async namespace deletion we could have a situation where the secrets are not yet deleted and we're trying to create a new mesh with the same secret name which k8s treats as changing the `mesh` of the secret https://app.circleci.com/pipelines/github/kumahq/kuma/24848/workflows/f33edb5a-74cb-45ae-b0c2-4b14bf289585/jobs/497239.

Reply:
Yeah makes sense to me